react-components/src/analysis/view/AnalysesView.js

```diff
@@ -352,21 +383,23 @@ class AnalysesView extends Component {
             const idFilterObj = Object.create(filter);
             idFilterObj.field = ID;
             idFilterObj.value = idFilter;
-            searchFilters.push(idFilterObj);
+            searchFilters.push(idFilterObj);           
         }
 
         return searchFilters;
 
     }
-    
+
 
     handleChangePage(event, page) {
         const {rowsPerPage} = this.state;
-        this.setState({page: page, offset: rowsPerPage * page});
+        //reset selection between pages
+        this.setState({page: page, offset: rowsPerPage * page, selected: []},
+            () => this.fetchAnalyses(false));
```

Comment:
No need for `false` here either.

Reply:
Ah ya. Forgot to clean up these ones from old code. Thanks for catching this 👍